hack/create-user-workspace.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+if [[ -z ${1} ]]
+then
+  echo "You have to provide the name of the service you want to initiate in the user workspace:"
+  echo "     create-user-workspace.sh <appstudio|hacbs>"
+  exit 1
+fi
+
+SERVICE_NAME=${1}
+
+ROOT_WORKSPACE=${ROOT_WORKSPACE:-"root"}
+APPSTUDIO_WORKSPACE=${APPSTUDIO_WORKSPACE:-"redhat-appstudio"}
+HACBS_WORKSPACE=${HACBS_WORKSPACE:-"redhat-hacbs"}
+
+if [ "${ROOT_WORKSPACE}" == "~" ]; then
+  ROOT_WORKSPACE=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl ws . | cut -f2 -d'"')
```

Comment:
Requires `KCP_KUBECONFIG`, other kubectl does not require it

Reply:
already solved in https://github.com/redhat-appstudio/infra-deployments/commit/91b5862b72bb262619c3187a61f891d9a3e2dd42